### PR TITLE
LIVE-3899 postgresql upgrade

### PR DESCRIPTION
**Pull request by waisingyiu**
## What does this change?

We are planning to perform database engine upgrade on the registrations database from Postgresql 10.18 to a newer version.

The documents in the PR summaries the study on the database upgrade, the recommendation, the detailed procedures and the test result.

### CODE REVIEW

File: docs/testing/06-postgresql-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+| # | Total harvester duration (s) | No. harvester invocations | Harvester error "marked as broken" |
+| ----------- | ----------- | ----------- | ----------- |
+1 | 33.41| 158 | 0 |
+2 | 14.52| 158 | 0 |
+3 | 17.94| 158 | 0 |
+4 | 24.6 | 158 | 0 |
+| AVG | 22.6175 | 158 | 0 |
+
+## Conclusion
```
Comment (groakland) [inline docs/testing/06-postgresql-upgrade.md:44]: Do we have a Conclusion, or shall we add this later

**Review by groakland — APPROVED** [review]
Looks great

**Review by frankie297 — APPROVED** [review]
looks great, thanks Wai Sing 👍

File: docs/testing/06-postgresql-upgrade.md (hunk 1 above)
Comment (waisingyiu) [reply docs/testing/06-postgresql-upgrade.md:44]: Apologies!  I have added the conclusion.

**Pull request merged**